src/main/java/test/TestMaksym/TestNewUser.java

```diff
@@ -1,57 +1,55 @@
+package main.java.test.TestMaksym;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Severity;
+import io.qameta.allure.SeverityLevel;
+import main.java.data.User;
+import main.java.data.UserRepository;
+import main.java.pages.login.LoginPage;
+import main.java.pages.main.MainPage;
+import main.java.pages.register.RegisterPage;
+import main.java.test.test;
+import main.java.tools.OurWebDriver;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+@Listeners (test.class)
+
+public class TestNewUser extends test {
+  
+  @Severity(SeverityLevel.CRITICAL)
+  @Description("Verify creation of the new user account")
+	@Test
+	public void testCreateNewUser() throws InterruptedException {
```

Comment:
fix identation

Reply:
tried to fix, commit via --amend and pushed,   but no changes appeared in this merge
if you don't mind let us merge this 'as is' and I will provide changes the next time